cpp/inc/bond/protocol/simple_json_reader.h

```diff
@@ -179,15 +188,88 @@ class SimpleJsonReader
     friend typename boost::enable_if<is_map_container<X> >::type
     DeserializeMap(X&, BondDataType, const T&, SimpleJsonReader<Buffer>&);
 
-    SimpleJsonReader(const SimpleJsonReader& that, const char* name)
-        : _stream(that._stream),
-          _document(that._document),
-          _value(&(*that._value)[name])
-    {}
+    /// @brief Holds either an input stream XOR a pointer to some parent
+    /// StreamHolder.
+    class StreamHolder
+    {
+    public:
+        explicit StreamHolder(const Buffer& input)
+            : _stream(),
```

Comment:
Skip both? ;)

Reply:
No, that makes that static analysis tool that reported the original bugs (in an internal bug tracking system) unhappy.